src/theme.js

```diff
@@ -0,0 +1,9 @@
+export const theme = {
+    color: {
+    white: "#FFFFFF",
+    black: "#000000",
+},
+breakpoint: {
+    mobile : 767
```

Comment:
value of object properties should be in quotes, colon on the end of the line would help to avoid possible errors

Reply:
sorry, I wrote "colon", but meant "comma" ;)

Reply:
added comma to breakpoint

Reply:
and added quotation marks in breakpoint